### PR TITLE
Add Privacy Policy, and links to AkitaFeedback@gmail.com

### PR DESCRIPTION
**Pull request by vezwork**
Part of #73

Adds a Privacy Policy as required by [Privacy Policy & Secure Handling Requirements - Chrome Developers](https://developer.chrome.com/docs/webstore/user_data/). The privacy policy addresses the following questions: 

- What information does Akita collect?
- How does Akita use the information it collects?
- What information does Akita share? (none)
- How long does Akita retain your data?
- Which browser permissions does Akita use?
- Where can I find Akita's source code?

Additionally updates links and references to the Privacy Policy in README.md, CONTRIBUTING.md, and in the last tutorial slide in the extension popup.

Additionally adds contact links to our new email AkitaFeedback@gmail.com to README.md and CONTRIBUTING.md

This PR was co-authored by @sharon-wang .

### CODE REVIEW

File: docs/PrivacyPolicy.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+This Privacy Policy applies to the Akita Browser Extension developed by the contributors at
+[https://github.com/esse-dev/akita](https://github.com/esse-dev/akita). This Privacy Policy
+addresses the following questions:
+
+1. [What information does Akita collect?](#1-What-information-does-Akita-collect?)
+2. [How does Akita use the information it collects?](#2-How-does-Akita-use-the-information-it-collects?)
+3. [What information does Akita share? (none)](#3-What-information-does-Akita-share?)
+4. [How long does Akita retain your data?](#4-How-long-does-Akita-retain-your-data?)
+5. [Which browser permissions does Akita use?](#5-Which-browser-permissions-does-Akita-use?)
+6. [Where can I find Akita's source code?](#6-Where-can-I-find-Akita's-source-code?)
```
Comment (sharon-wang) [inline docs/PrivacyPolicy.md]: These links don't work. Remove the `?` from each and remove the `'` from the 6th item. Markdown doesn't include punctuation in its links

File: docs/PrivacyPolicy.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
+# Akita Privacy Policy
+
```
Comment (sharon-wang) [inline docs/PrivacyPolicy.md:2]: Include a "last updated" line with today's date

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -24,12 +25,13 @@ We're always looking to make Akita better, so definitely drop us a Feature Reque
 if you have any ideas brewing!
 
 Issue Templates you can choose from:
+
 - 📄 [Create New Blank Issue](https://github.com/esse-dev/akita/issues/new?assignees=&labels=&template=blank-issue.md&title=)
 - 🐞 [Create New Bug Report](https://github.com/esse-dev/akita/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBUG%5D)
 - 💡 [Create New Feature Request](https://github.com/esse-dev/akita/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=%5BFEATURE+REQUEST%5D)
 
 ## Contribute Code
-
```
Comment (sharon-wang) [inline CONTRIBUTING.md:32]: I think it'd be good to have a line in here to read the Privacy Policy before contributing code (so developers understand that we want to uphold privacy!) and so they'll know to update the Privacy Policy if any code changes require updating the Privacy Policy

**Review by sharon-wang — CHANGES_REQUESTED** [review]
Looks good, just some fixes/additions requested. Other than the broken links in the PrivacyPolicy contents section, I tested all the other links in the doc and they're working.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -24,12 +25,16 @@ We're always looking to make Akita better, so definitely drop us a Feature Reque
...
 - 📄 [Create New Blank Issue](https://github.com/esse-dev/akita/issues/new?assignees=&labels=&template=blank-issue.md&title=)
 - 🐞 [Create New Bug Report](https://github.com/esse-dev/akita/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBUG%5D)
 - 💡 [Create New Feature Request](https://github.com/esse-dev/akita/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=%5BFEATURE+REQUEST%5D)
 
 ## Contribute Code
-	 
+
+Before contributing code to Akita please read [Akita's Privacy Policy](docs/PrivacyPolicy.md) so you
+can help us ensure data privacy for Akita users! If you make any changes that affect user data,
+please update the Privacy Policy as part of your changes.
```
Comment (sharon-wang) [inline CONTRIBUTING.md]: > please update the Privacy Policy as part of your changes.

suggested reword: `please review the Privacy Policy to check if any updates should be made to it alongside your changes`

and a nit: newline between this text and `### Implement Your Changes`

**Review by sharon-wang — CHANGES_REQUESTED** [review]

**Comment by vezwork** [comment]
Thanks! Addressed your comments.

**Review by sharon-wang — APPROVED** [review]
Awesome, looks good! Just missing a summary of the changes in the PR, otherwise ready to go.

**Pull request merged**